web/styling/box-model/borders.md

```diff
@@ -36,15 +38,19 @@ div{
 }
 ```
 
-Or using the shorthand notation, that respects the same order:
+Or using the shorthand notation:
 
 ```css
 div{
-  border: 4px solid red;
+  border: red 4px solid;
 }
 ```
 
-Borders can have different styles, of which the most common is `solid`. Others are `dashed`,`dotted` or `double`.
+You can declare it with any combination of the `width`, `style`, and `color`, as well as in any order.
+
+> ❗ The `style` needs to be set for the border to show. It is because the default value is `none`. 
+
+Borders can have different styles, of which the most common is `solid`. Others are `dashed`, `dotted`, or `double`.
```

Comment:
imo we can make this one phrase

if there are no other styles, remove the ", and others" at the end of the phrase

```suggestion
Borders can have different styles such as `solid` (**the most common**), `dashed`, `dotted`, `double`, and others.
```

Reply:
Yeah, there are a few 
border-style: none;
border-style: hidden;
border-style: dotted;
border-style: dashed;
border-style: solid;
border-style: double;
border-style: groove;
border-style: ridge;
border-style: inset;
border-style: outset;